t/151-exit-worker.t

```diff
@@ -0,0 +1,60 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket 'no_plan';
+use Test::Nginx::Socket::Lua;
+
+
+#log_level("warn");
+no_long_string();
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: simple exit_worker_by_lua_block
+--- http_config
+    exit_worker_by_lua_block {
+        ngx.log(ngx.NOTICE, "log from exit_worker_by_lua_block")
```

Comment:
@lixianliang Apparently `iresty/test-nginx` is a fork and does not count here :) That feature needs to be merged into the official `test-nginx` repo before this can be merged otherwise the Travis CI would be failing.